include/curl/curl.h

```diff
@@ -37,6 +37,15 @@
 #include "curlver.h"         /* libcurl version defines   */
 #include "system.h"          /* determine things run-time */
 
+/*
+ * Define CURL_WIN32 when build target is Win32 API
+ */
+
+#if (defined(_WIN32) || defined(__WIN32__)) &&  \
+  !defined(WIN32) && !defined(__SYMBIAN32__)
```

Comment:
I think the `!defined(WIN32)` is not necessary anymore. Probably just used to be there to avoid a compiler warning about `WIN32` being already defined?

Reply:
Yeah, that was just me bringing back the old logic from before without proper polish.... will fixup.